hatch/client.py

```diff
@@ -35,7 +35,12 @@ def fetch_index(token, workspace, release_id=None):
         kwargs["release_id"] = release_id
     else:
         view_name = "workspace_index"
-    resp = requests.get(app.reverse_url(view_name, **kwargs), headers=headers)
+
+    # generate url
+    path = app.app.url_path_for(view_name, **kwargs)
```

Comment:
`app.app.url_path_for` looks like a typo is that correct??

Reply:
(this may be a stupid question...)

Reply:
So, `hatch.app`  is a module, and `hatch.app.app` is an instance of a FastAPI app.

Not great naming

Reply:
ok then